test/src/tests/e2e/sitegen/page-objects/product.js

```diff
@@ -0,0 +1,80 @@
+import { expect } from '@playwright/test'
+import path from 'path'
+import { BasePage } from './base'
+import { AccountPage } from './account'
+import * as dotenv from 'dotenv'
+
+dotenv.config({ path: path.join('..', '.env') })
+
+exports.ProductPage = class ProductPage extends BasePage {
+    constructor(page, isMobile) {
+        super(page, isMobile)
+
+        this.accountPage = new AccountPage(page, isMobile)
+
+        this.newArrivals = page.locator('.has-sub-menu')
+        this.categoryLocator = page.locator('.category-tile')
+        this.productLocator = page.locator('.thumb-link')
+        this.productTitle = page.locator('.product-name')
+
+        this.sizeSwatches = page.locator('.swatches.size')
+        this.sizeSelect = page.locator('.swatchanchor')
+        // this.widthSelect = page.locator('#width-1')
```

Comment:
Are the commented out lines in this file still needed?

Reply:
@cykolln for what we need for these tests, this commented code can be removed. I will update and make a PR.